Dockerfile

```diff
@@ -1,40 +1,39 @@
-FROM amazonlinux:2.0.20200722.0
+FROM ubuntu:focal-20210401
 
+ARG DEBIAN_FRONTEND=noninteractive
 ARG STACK_VERSION=2.5.1
-ARG USER=haskell
 
 ENV LANG=C.UTF-8
-ENV PATH="/home/$USER/.local/bin:$PATH"
 
-RUN yum update -y \
-  && amazon-linux-extras install postgresql11 \
-  && yum install -y \
+# Make sure this path includes initdb for the usage of tmp-postgres
+ENV PATH=/usr/lib/postgresql/12/bin/$PATH
+
+RUN apt-get update -y \
+  && apt-get install -y \
+    curl \
     gcc \
     git \
-    gmp-devel \
     gzip \
+    libgmp-dev \
+    liblzma-dev \
+    libncurses5-dev \
+    libpq-dev \
     make \
-    nc \
-    ncurses-devel \
+    netcat-openbsd \
     perl \
+    postgresql-12 \
```

Comment:
It's possible we can tighten this dependency down to just whatever `tmp-postgres` requires, but it's very difficult to figure out the exact package and we've sunk too much time to make it worth it. The only issue installing all of psql is that we'll have a server running for no reason in the dev toolbox.